badgr-fullstack-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,282 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "badgruser",
+      "metadata": {
+        "description": "Administrator username."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password."
+      }
+    },
+	"badgrSuperUserEmail": {
```

Comment:
run this file through json formatting (e.g. Alt+Shift+F in VS Code) to correct spacing